po/ja.po

```diff
@@ -18407,46 +18439,59 @@ msgid ""
 "[`PoisonError`](https://doc.rust-lang.org/std/sync/struct.PoisonError.html). "
 "You can call `into_inner()` on the error to recover the data regardless."
 msgstr ""
+"スレッドにパニックを起こした`Mutex`がある場合、保護すべきデータが整合性の欠け"
```

Comment:
```suggestion
"`Mutex`を保持したスレッドがパニックを起こした場合、保護すべきデータが整合性の欠け"
```

Reply:
In your original translation, it can be read that a Mutex can cause panick, but it doesn't make sense as Mutex is a value.